mailpoet/tests/integration/AutomaticEmails/WooCommerce/Events/AbandonedCartTest.php

```diff
@@ -269,6 +272,9 @@ public function testItSchedulesNewEmailWhenEmailAlreadySent() {
     $this->assertEquals($scheduled->getScheduledAt(), $expectedTime);
   }
 
+  /**
+   * @group woo
```

Comment:
Aha. Nice catch! This is leftover. I'll remove it. I'll try to explain how it happened.
When I started adding the groups, I thought that I would add it on the test case level and pick only tests that really call some function from the WooCommerce codebase.
But then I realized that this is not how the semantics of the group works or, in other words, that the better semantics for the group would be that the tested functionality is related to Woo integration.

So it is still ok to mark just one test case or service that is just partially related to Woo integration. But in this case, the whole class was made because of Woo integration, so I marked the whole test class, but I forgot to remove the group from the test case I marked before.